pkgs/top-level/all-packages.nix

```diff
@@ -11764,6 +11764,10 @@ let
 
   rtl-sdr = callPackage ../applications/misc/rtl-sdr { };
 
+  rtv = callPackage ../applications/misc/rtv {
+    inherit (pythonPackages) requests six praw;
```

Comment:
This and `requests, six, praw` in the 1st line of `default.nix` are extra since you are using `propagatedBuildInputs = with pythonPackages;`.